README.md

```diff
@@ -28,6 +28,6 @@ Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
 
 # Answer
-The slides state that the recurrence relation is $T(n) = 2 * T(n/2) + 2$ for a merge sort splitting into two parts. When we split into three parts the recurrance relation simply becomes $T(n) = 3 * T(n/3) + 3$ which we then follows the substitution steps to get $T(n) = 3 * (3 * T(n/3/3) + 3) + 3$ which simplifies to $T(n) = 9 * T(n/9) + 6$.
+The slides state that the recurrence relation is $T(n) = 2 * T(n/2)$ for a merge sort splitting into two parts. When we split into three parts the recurrance relation simply becomes $T(n) = 3 * T(n/3)$ which we then follows the substitution steps to get $T(n) = 3 * (3 * T(n/3/3))$ which simplifies to $T(n) = 9 * T(n/9)$.
 
-To get our $\Theta$ we can take our $T(n) = 3 * T(n/3) + 3$ and plug in $i$ to get $3^i * T(n/3^i) + 3i$ where $i$ is $i = log3(n)$. We can then simplify this to get $n * T(1) + 3log(n) = n(1) + 3 * log(n) = n * 3log(n)$ to get our final answer of $\Theta = n * 3log(n)$
+To get our $\Theta$ we can take our $T(n) = 3 * T(n/3)$ and plug in $i$ to get $3^i * T(n/3^i)$ where $i$ is $i = log3(n)$. We can then simplify this to get $n * T(1)$ which just gives us $n$ we can then use this to get our final answer of $\Theta(n)$
```

Comment:
You say that for merge sort we have T(n) = 2T(n/2)! Where is the combine part? Every divid-and-conquer relation should be written as T(n) = aT(n/b) + f(n); f(n) can be constant. Note: For merge-sort, the recurrence relation is: T(n) = 2T(n/2) + n. According to this definition, you need to change the recurrence relation for this particular example.